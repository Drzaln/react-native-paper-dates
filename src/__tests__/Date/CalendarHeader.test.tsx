import { render } from '@testing-library/react-native'
import CalendarHeader from '../../Date/CalendarHeader'

it('renders CalendarHeader', () => {
  const { toJSON } = render(
    <CalendarHeader
      locale="en"
      onPrev={() => null}
      onNext={() => null}
      scrollMode="vertical"
      startWeekOnMonday={false}
    />
  )
  expect(toJSON()).toMatchSnapshot()
})
