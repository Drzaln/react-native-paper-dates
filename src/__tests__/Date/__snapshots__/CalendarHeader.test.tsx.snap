// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CalendarHeader 1`] = `null`;
